packages/orbit-tailwind-preset/src/presets/components/index.ts

```diff
@@ -250,6 +264,8 @@ const cfg = (options?: Options): Config => {
           "form-element-hover": tokens.formElementBoxShadowHover,
           "form-element-focus": tokens.formElementFocusBoxShadow,
           switch: `inset 0 0 1px 0 rgba(7, 64, 92, 0.1),${tokens.boxShadowAction}`,
+          "modal-scrolled": `inset 0 1px 0 ${tokens.paletteCloudNormal}, ${tokens.boxShadowFixedReverse}`,
+          modal: `inset 0 0 0 transparent, ${tokens.boxShadowFixedReverse}`,
```

Comment:
```suggestion
          elevation-flat: tokens.elevationFlatBorderColor,
```

Reply:
I don't understand this suggestion 😕 can you please explain?

Reply:
ah, I thought it's border color, np, ignore this one then :)